source/intro.tex

```diff
@@ -928,6 +939,17 @@
 is an observable checkpoint.
 \end{note}
 
+\pnum
+\indextext{observable checkpoint}
+Certain events in the execution of a program
+are termed \defnadj{observable}{checkpoints}.
+\begin{note}
+A call to \tcode{std::observable}\iref{utility.undefined}
+is an observable checkpoint,
```

Comment:
This is duplicate with the preceding paragraph.  Somehow, this needs to be rebased properly.

Reply:
boo on git.  fixing.